docker/generate_dockerfiles.sh

```diff
@@ -53,40 +53,40 @@ do
 done
 
 
-# neurodocker version 0.3.1-22-gb0ee069
-NEURODOCKER_IMAGE="kaczmarj/neurodocker@sha256:f15ca90803f4b89acfca55cd1c7269bf2ec2dfd95b3de1118b08afa34b87d9d1"
+# neurodocker:master pulled on October 9, 2018
+NEURODOCKER_IMAGE="kaczmarj/neurodocker@sha256:c5c10fc11cbd7efe18ac10d2370742b8ecb3588c3aa9b2c15ca7e18363bfa9dc"
 
-# neurodebian:stretch-non-free pulled on November 3, 2017
-BASE_IMAGE="neurodebian@sha256:7590552afd0e7a481a33314724ae27f76ccedd05ffd7ac06ec38638872427b9b"
+# neurodebian:stretch-non-free pulled on October 9, 2018
+BASE_IMAGE="neurodebian@sha256:7cd978427d7ad215834fee221d0536ed7825b3cddebc481eba2d792dfc2f7332"
 
 NIPYPE_BASE_IMAGE="nipype/nipype:base"
 PKG_MANAGER="apt"
 DIR="$(dirname "$0")"
 
 function generate_base_dockerfile() {
-  docker run --rm "$NEURODOCKER_IMAGE" generate \
+  docker run --rm "$NEURODOCKER_IMAGE" generate docker \
   --base "$BASE_IMAGE" --pkg-manager "$PKG_MANAGER" \
   --label maintainer="The nipype developers https://github.com/nipy/nipype" \
-  --spm version=12 matlab_version=R2017a \
+  --spm12 version=r7219 \
   --afni version=latest install_python2=true \
   --freesurfer version=6.0.0 min=true \
   --run 'echo "cHJpbnRmICJrcnp5c3p0b2YuZ29yZ29sZXdza2lAZ21haWwuY29tXG41MTcyXG4gKkN2dW12RVYzelRmZ1xuRlM1Si8yYzFhZ2c0RVxuIiA+IC9vcHQvZnJlZXN1cmZlci9saWNlbnNlLnR4dAo=" | base64 -d | sh' \
   --install ants apt-utils bzip2 convert3d file fsl-core fsl-mni152-templates \
             fusefat g++ git graphviz make ruby unzip xvfb \
   --add-to-entrypoint "source /etc/fsl/fsl.sh" \
   --env ANTSPATH='/usr/lib/ants' PATH='/usr/lib/ants:$PATH' \
-  --run "gem install fakes3" \
-  --no-check-urls > "$DIR/Dockerfile.base"
+  --run "gem install fakes3" > "$DIR/Dockerfile.base"
 }
 
 
 function generate_main_dockerfile() {
-  docker run --rm "$NEURODOCKER_IMAGE" generate \
+  docker run --rm "$NEURODOCKER_IMAGE" generate docker \
   --base "$NIPYPE_BASE_IMAGE" --pkg-manager "$PKG_MANAGER" \
   --label maintainer="The nipype developers https://github.com/nipy/nipype" \
   --env MKL_NUM_THREADS=1 OMP_NUM_THREADS=1 \
   --user neuro \
-  --miniconda env_name=neuro \
+  --miniconda create_env=neuro \
+              conda_install='python=${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR}' \
```

Comment:
Looks like this is being used prior to the definition of `PYTHON_VERSION_*` (L102).